client/src/actions/teaActions.js

```diff
@@ -25,7 +24,13 @@ export function addTea(tea) {
           type: "ADD_TEA",
           payload: async (_action, _state, res) => {
             res = await res.json();
-            return res;
+            return {
+              id: res.id,
+              name: res.name,
+              brand: res.brand,
+              teaType: res.teaType,
+              servings: res.servings
+            };
```

Comment:
Write a helper function that returns this object of required fields so you don't have to do it three times in this reducer.

Reply:
i was going to do this during the refactor to remove the RSAA and replace it with the internal api.

Reply:
Might as well do it now. No reason to introduce technical debt.